examples/compiled/interactive_line_brush_cursor.vg.json

```diff
@@ -259,7 +259,7 @@
       "clip": true,
       "encode": {
         "enter": {
-          "cursor": {"value": "pointer"},
```

Comment:
Why is this one changing? Seems odd that there was one already and you override it.